config/hh5p.php

```diff
@@ -20,11 +20,11 @@
 
     'h5p_show_display_option'    => true,
     'h5p_frame'                  => true,
-    'h5p_export'                 => false,
+    'h5p_export'                 => true,
```

Comment:
```suggestion
    'h5p_export'                 => false,
```

this should be `false` by default